src/Foundation/Http/Controllers/Tools/FileFunctionsService.php

```diff
@@ -142,9 +148,10 @@ private static function checkFileExists($folder, $name)
      *
      * @param $path_to_png_file string - path to any PNG file, e.g. $_FILE['file']['tmp_name']
      * @param $max_quality int - conversion quality, useful values from 60 to 100 (smaller number = smaller file)
+     *
      * @return string - content of PNG file after conversion
      */
-    static function compress_png($path_to_png_file, $max_quality = 90)
+    public static function compress_png($path_to_png_file, $max_quality = 90)
```

Comment:
__[code_sniffer]__
* Method name "FileFunctionsService::compress_png" is not in camel caps format

<!-- Comment by [SideCI](https://sideci.com) -->